src/components/UserManagement/UserTableData.jsx

```diff
@@ -111,6 +121,7 @@ const UserTableData = React.memo(props => {
           className={`btn btn-outline-primary btn-sm${
             props.timeOffRequests?.length > 0 ? ` time-off-request-btn-moved` : ''
           }`}
+          disabled={isOwner? false : true}
```

Comment:
It looks like this button is for requesting time off. Is that related to your PR??